docs/data-ai/capture-data/lorawan.md

```diff
@@ -0,0 +1,522 @@
+
+- `viam:lorawan:dragino-LHT65N`: Dragino LHT65N temperature and humidity sensor.
+- `viam:lorawan:dragino-WQSLB`: Dragino WQS-LB water quality sensor
+- `viam:lorawan:milesight-ct101`: Milesight CT101 current sensor
+- `viam:lorawan:milesight-em310-tilt`: Milesight EM310-TILT sensor
+- `viam:lorawan:node`: Any LoRaWAN sensor that is:
+  - Class A
+  - supports either the `US915` or `EU868` frequency band
+  - uses LoRaWAN MAC specification version 1.0.3
+
+Device-specific models for Dragino and Milesight sensors provide default values for `app_key`, `network_s_key`, and `app_s_key`, so you can omit those fields from your configuration.
+
+Configure attributes based on the tables below:
+
+{{% /tab %}}
+{{< /tabs >}}
+
+You must configure the following attributes for OTAA nodes:
+
+<!-- prettier-ignore -->
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| `dev_addr` | string | Optional | 32-bit hexadecimal **device address** used to identify this device in uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. |
+| `app_s_key` | string | Optional | 128-bit hexadecimal **application session key** used to decrypt uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. <br> On `milesight-ct101` and `milesight-em310-tilt`, defaults to `5572404C696E6B4C6F52613230313823`. |
+| `network_s_key` | string | Optional | 128-bit hexadecimal **network session key** used to decrypt uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. <br> On `milesight-ct101` and `milesight-em310-tilt`, defaults to `5572404C696E6B4C6F52613230313823`. |
+| `gateways` | string[] | **Required** | Name of the [gateway component](#add-a-gateway) in your Viam configuration. |
+
+You must configure the following attributes for ABP nodes:
+
+<!-- prettier-ignore -->
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| `join_type` | string | Optional | The [activation protocol](#activation-protocols) used to secure this network. <br> Default: "OTAA". <br> Options: "OTAA", "ABP" |
+| `dev_eui` | string | **Required** | The **device EUI (Extended Unique Identifier)**, a unique 64-bit identifier for the LoRaWAN device in hexadecimal format (16 characters). Found on your device or in device packaging. |
+| `app_key` | string | Optional | The 128-bit hexadecimal AES **application key** used for device authentication and session key derivation. Required for OTAA activation protocol. Found in the device datasheet. <br> On `milesight-ct101` and `milesight-em310-tilt`, defaults to `5572404C696E6B4C6F52613230313823`. |
+| `gateways` | string[] | **Required** | Name of the [gateway component](#add-a-gateway) in your Viam configuration. |
+
+For generic nodes, you must also configure a [`decoder_path`](#decoder-script).
```

Comment:
what are generic nodes? 

Also the connection between decoder_path and what it should link to is a bit missing:
```suggestion
For generic nodes, you must also configure a `decoder_path` for the [decoder script](#decoder-script).
```